### ISSUE TITLE
Data migrate failing on node v12.19.0 (Affects Netlify and Vercel Deploy)

### ISSUE THREAD

**Issue by dac09** [redwoodjs/redwood#1313]
### What?
As of Redwood v19.x, data migrations are failing due to a low level Node.js error when running on Node v12.19.0

```
./src/node_http_parser_impl.h:529:static void node::(anonymous namespace)::Parser::Initialize(const FunctionCallbackInfo<v8::Value> &): Assertion `args[3]->IsInt32()' failed.
 1: 0x1011f40c5 node::Abort() (.cold.1) [/Users/dac09/.nvm/versions/node/v12.19.0/bin/node]
 2: 0x1000a5749 node::Abort() [/Users/dac09/.nvm/versions/node/v12.19.0/bin/node]
 3: 0x1000a55b1 node::Assert(node::AssertionInfo const&) [/Users/dac09/.nvm/versions/node/v12.19.0/bin/node]
 4: 0x1000c6e1b node::(anonymous namespace)::Parser::Initialize(v8::FunctionCallbackInfo<v8::Value> const&) [/Users/dac09/.nvm/versions/node/v12.19.0/bin/node]
 5: 0x100259838 v8::internal::FunctionCallbackArguments::Call(v8::internal::CallHandlerInfo) [/Users/dac09/.nvm/versions/node/v12.19.0/bin/node]
 6: 0x100258df9 v8::internal::MaybeHandle<v8::internal::Object> v8::internal::(anonymous namespace)::HandleApiCallHelper<false>(v8::internal::Isolate*, v8::internal::Handle<v8::internal::HeapObject>, v8::internal::Handle<v8::internal::HeapObject>, v8::internal::Handle<v8::internal::FunctionTemplateInfo>, v8::internal::Handle<v8::internal::Object>, v8::internal::BuiltinArguments) [/Users/dac09/.nvm/versions/node/v12.19.0/bin/node]
 7: 0x100258562 v8::internal::Builtin_Impl_HandleApiCall(v8::internal::BuiltinArguments, v8::internal::Isolate*) [/Users/dac09/.nvm/versions/node/v12.19.0/bin/node]
```

This is a problem because Netlify uses 12.9 by default for builds.

### How to reproduce?
Run ```yarn rw dataMigrate up``` if you're running Node 12.9. Or simply push to netlify to build, as long as data migrate is part of the build command

### Workaround
The issue seems to be in prisma client > undici > node 12.9.0

Easy fix for now is to downgrade to node 12.8 OR upgrade to node stable (currrently 14).

In mycase, ~~I changed the value in .nvmrc to "stable"~~ (note: "stable" has been deprecated. See [this comment](https://github.com/redwoodjs/redwood/issues/1313#issuecomment-706443974) for workaround.)

### Todo

- [x] Reproduce on other projects
- [x] Identify where the issue is coming from
- [x] Propose workaround

**Comment by dac09** [comment]
So the issue is coming from trying to import the prisma client. Something's changed since v18 in prisma, which means its not working with babel correctly. 

I've removed this line [here](https://github.com/redwoodjs/redwood/blob/4191be9bf1d7b791f13106978bde990d132eb3e9/packages/cli/src/commands/dataMigrate/up.js#L12), and it seems to fix the obscure error, but (obviously) fails because db isn't present.

**Comment by dac09** [comment]
Found relevant prisma issue: https://github.com/prisma/prisma-client-js/issues/907

**Comment by thedavidprice** [comment]
@dac09 can you confirm v0.19.2 deploys successfully on Netlify if you use NodeJS v14+ for the build, correct?

If so, could be a workaround for people in the meantime.

**Comment by thedavidprice** [comment]
This will resolve when either NodeJS v12.19.1 is released **or** Prisma v2.9.0 is released + Redwood upgraded.

## Workaround
Pin the version of NodeJS used for Netlify and Vercel build in your `.nvmrc` file:
- Replace
```
// .nvmrc

lts/*
```
- with
```
// .nvmrc

v12.18.4
```
